streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -1516,6 +1488,27 @@ object ZStream extends ZStreamPlatformSpecific {
   final def bracket[R, E, A](acquire: ZIO[R, E, A])(release: A => ZIO[R, Nothing, _]): ZStream[R, E, A] =
     managed(ZManaged.make(acquire)(release))
 
+  final def combine[R, E, A, B, C](s1: ZStream[R, E, A], s2: ZStream[R, E, B], lc: Int = 2, rc: Int = 2)(
```

Comment:
This one could also use an infix version.

Reply:
It's also interesting to think of how would the generalized n-stream version would look like!

Reply:
Is it better to just do everything infix? That seems more consistent with most of the other methods.

Reply:
Yes, good point. Probably only variadic combinators should have a prefix form.

Reply:
And yes about generalizing further to multiple streams! But that I think we can save for another PR! 😃